cfgov/unprocessed/css/schema.less

```diff
@@ -2,10 +2,6 @@
     margin-top: unit( @grid_gutter-width / @size-ii, em );
 }
 
-.schema-block .rich-text h3 {
-    margin-top: unit( @grid_gutter-width / @size-iii, em );
-}
-
```

Comment:
Note to self: Check on this one, too. Might've just needed the `.rich-text` removed from the selector.

Reply:
This is also fine.